.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -20,8 +20,15 @@ A clear and concise description of what you expected to see.
 A clear and concise description of what you saw instead.
 
 **What version and what artifacts are you using?**
+Artifacts: (e.g., `opentelemetry-api`, `opentelemetry-sdk`, which exporters, etc)
 Version: (e.g., `v0.4.0`, `1eb551b`, etc)
-Artifacts: (e.g., `api`, `sdk`, etc)
+How did you reference these artifacts? (excerpt from your `build.gradle`, `pom.xml`, etc)
```

Comment:
Are you sure we want to use `excerpt` here ?

Reply:
Yes, otherwise the report could get pretty bloated. Would you prefer getting the full build config? If they provide a minimal repro it should probably include a complete but stripped-down build config.